src/ConcreteOperations/intersection.jl

```diff
@@ -676,21 +676,44 @@ Compute the intersection of two polytopes in vertex representation.
 
 ### Output
 
-A `VPolygon` if both arguments are `VPolygon`s, and a `VPolytope` otherwise.
+A `VPolytope`.
 """
-function intersection(P1::Union{VPolytope{N}, VPolygon{N}},
-                      P2::Union{VPolytope{N}, VPolygon{N}};
+function intersection(P1::VPolytope{N},
+                      P2::VPolytope{N};
```

Comment:
Was it necessary to downgrade this method? Because I think we now lost the intersection for mixed types. I would expect that Julia would use your other method automatically because it is more specific.

Reply:
i think that for mixed types they should use `_intersection_vrep`, i can try changing that if u agree. 

how about VPolytope's in 2D? method `_intersection_vrep` is much faster (from 1-2us to 1-2ms) so i guess we it's also OK to use the method for polygons in that case? and return a VPolytope.

Reply:
> i can try changing that if u agree

:+1:

> how about VPolytope's in 2D?

as you prefer